src/features/VoltageConfiguration/VoltageConfiguration.tsx

```diff
@@ -40,6 +40,15 @@ const VoltageConfiguration = ({
         pmicPortDescription ??
         `Set voltage for PMIC port ${pmicPort} (${label})`;
 
+    // Voltage presets in lieu of presets in board definition files
+    const voltagePresetValues = [1200, 1800, 2000, 3300, 1600, 2400]
+        .filter(
+            filterVoltage =>
+                filterVoltage >= voltageMin && filterVoltage <= voltageMax
+        )
+        .slice(0, 3)
+        .sort((a, b) => a - b);
```

Comment:
Not that it really matters, but why have unsorted presets only to explicitly sort them after?

Reply:
It does, in fact, matter :) The reason is to pick relevant voltage levels fro the different voltage ranges. I slice of the tree first elements to pick the most relevant fitting the range, and finally sort them to make the UI nice.

Good catch, though! :+1: